front/src/applications/editor/components/EntitySumUp.tsx

```diff
@@ -284,8 +287,8 @@ const EntitySumUp: FC<
   } & (
     | { entity: EditorEntity; id?: undefined; objType?: undefined }
     | { id: string; objType: EditoastType; entity?: undefined }
-  )
-> = ({ entity, id, objType, classes, status }) => {
+  ) & { error?: InfraError['information'] }
```

Comment:
Move the error above, after status
```suggestion
{
    classes?: Partial<typeof DEFAULT_CLASSES>;
    status?: string;
    error?: InfraError['information'];
  } & (
    | { entity: EditorEntity; id?: undefined; objType?: undefined }
    | { id: string; objType: EditoastType; entity?: undefined }
  )
```